01-node-tutorial/answers/11-fs-async.js

```diff
@@ -0,0 +1,30 @@
+const {readFile, writeFile} = require('fs')
+
+console.log('start')
+readFile('./content/first.txt', 'utf8', (err, result) => {
+    if (err) {
+        console.log(err)
+        return
+    }
+    const first = result
+    readFile('./content/second.txt', 'utf8', (err, result) => {
+        if (err) {
+            console.log(err)
+            return
+        }
+        const second = result
+        writeFile(
+            './content/result-async.txt',
+            `Here is the result :  ${first}, ${second}`,
+            {flag: 'a'},
```

Comment:
Great practice using the flag, although it is not a necessity in this example. 

In the Node.js docs under filesystem you will find a ton of useful info on flags and what they do when it comes to reading and writing files.